globus_sdk/exc.py

```diff
@@ -133,6 +134,30 @@ def _load_from_json(self, data):
         self.request_id = data["request_id"]
 
 
+class AuthAPIError(GlobusAPIError):
+    """
+    Error class for the API components of Globus Auth.
+
+    Customizes JSON parsing.
+    """
+    def _load_from_json(self, data):
+        """
+        Load error data from a JSON document.
+
+        Sets `code` statically because Auth doesn't have API error codes, and
+        looks for a top-level "error" attribute.
+        """
+        self.code = "Error"
```

Comment:
Judging by the tests, this was being set to the Requests `reason` field? I don't see that done explicitly though, and it makes me nervous.
@aaschaer, would you be able to update the tests? That would be great.